article/app/views/fragments/articleBodyGarnett.scala.html

```diff
@@ -52,9 +53,12 @@
 
 
                         @if(!isPaidContent) {
-                            @if(article.metadata.designType.nameOrDefault == "comment" ||
+                            @if(!article.content.isColumn &&
+                                article.metadata.designType.nameOrDefault == "comment" ||
                                 article.metadata.designType.nameOrDefault == "guardianview") {
                                 @fragments.articleHeaderCommentGarnett(article, model, amp = amp)
+                            } else if(article.content.isColumn) {
+                                @fragments.articleHeaderColumn(article, model, amp = amp)
```

Comment:
If we move this branch to the top, we wouldn't need to check for `!article.content.isColumn` in the `articleHeaderCommentGarnett` branch